src/core/lib/security/authorization/cel_evaluation_engine.cc

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <grpc/support/port_platform.h>
+
+#include "src/core/lib/security/authorization/cel_evaluation_engine.h"
+
+std::unique_ptr<CelEvaluationEngine>
+CelEvaluationEngine::CreateCelEvaluationEngine(
+    const std::vector<envoy_config_rbac_v2_RBAC*>& rbac_policies) {
+  if (rbac_policies.size() < 1 || rbac_policies.size() > 2) {
+    gpr_log(GPR_ERROR,
+            "The rbac_policies vector does not have 1 or 2 policies.");
```

Comment:
Since this is a log statement, lets not use variable names.
"Invalid rbac policies vector. Must contain either one or two rbac policies."
Similarly below.